sdk/appcenter-auth/src/test/java/com/microsoft/appcenter/auth/AuthTest.java

```diff
@@ -284,17 +304,45 @@ private void testInvalidConfig(JSONObject jsonConfig) throws Exception {
         Auth auth = Auth.getInstance();
         start(auth);
 
+        /* Mock public client application. */
+        mockMsalPublicClientApplication();
+
+        /* When we get a payload valid for AppCenter fields but invalid for msal ones. */
+        mockSuccessfulHttpCall(jsonConfig, mHttpClient);
+
+        /* We saved after we downloaded the file. */
+        verifyStatic();
+        FileManager.write(any(File.class), anyString());
+
+        /* We cleared the cache since the file was invalid. */
+        verifyStatic();
+        FileManager.delete(any(File.class));
+    }
+
+    private void testValidConfig(JSONObject jsonConfig) throws Exception {
+
+        /* Start auth service. */
+        Auth auth = Auth.getInstance();
+        start(auth);
+
+        /* mock public client application for msal*/
```

Comment:
```suggestion
        /* Mock public client application for MSAL*/
```

To capitalize MSAL as it’s an acronym.